sdk/storage/Azure.Storage.Common/swagger/Generator/src/models.ts

```diff
@@ -237,7 +238,8 @@ export interface IResponse {
     model?: IModelType,
     exception?: boolean,
     public: boolean,
-    returnStream?: boolean
+    returnStream?: boolean,
+    struct: boolean
```

Comment:
Does the response need a `struct` property?  Can't you just set it on `model` in `createResponse`?

Reply:
> Can't you just set it on model in `createResponse`?

How will we set `struct` in [createResponseType](https://github.com/Azure/azure-sdk-for-net/blob/9ca9e5b302baeb2556013389c935fc034d005ee5/sdk/storage/Azure.Storage.Common/swagger/Generator/src/serviceModel.ts#L963)?